lib/rpmsg/rpmsg_virtio.c

```diff
@@ -574,6 +574,8 @@ static int rpmsg_virtio_ns_callback(struct rpmsg_endpoint *ept, void *data,
 		} else {
 			_ept->dest_addr = dest;
 			metal_mutex_release(&rdev->lock);
+			if (_ept->ns_bound_cb)
```

Comment:
Put a comment here to say what it's doing, something like:

Notify application that the endpoint has been bound.

Reply:
Done.